cli/index.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+import fs from "fs";
+import "isomorphic-fetch";
+import { Command } from "commander";
+import inquirer from "inquirer";
+import chalk from "chalk";
+import asciify from "asciify-image";
+import chalklet from "chalklet";
+import ItemManager from "./item-manager.js";
+import PokemonClient from "./pokemon-client.js";
+
+const POKEMON_FILE_NAME = "pokemon-names.json";
+const TASKS_FILE_NAME = "tasks.json";
+const tasksManager = new ItemManager(render);
+if (!fs.existsSync(TASKS_FILE_NAME)) {
```

Comment:
File reading and writing shoule be done in another file. the cli should only handle the user input and call the relevant logic

Reply:
consider create a new file handling the reading and writing from the file.
in ItemManager you should "wrap" this logic and call this class in order to add or remove todos
from the cli you should only parse the user input and call the relevant method in item manager

Reply:
I just moved the file handling code to the same file as the business logic. 
Since I use the ItemManager class in the web app as well (without writing to a file) then I'm not sure if I should add the reading/writing wrapping to it.